stl/inc/vector

```diff
@@ -3859,7 +3859,25 @@ _CONSTEXPR20 _OutIt _Copy_vbool(_VbIt _First, _VbIt _Last, _OutIt _Dest) {
     }
 
 #if _HAS_CXX20
-    if (!_STD is_constant_evaluated())
+    if (_STD is_constant_evaluated()) {
+        if (_Dest._Myoff == _First._Myoff) {
+            const auto _FirstSourceVal = *_VbFirst & _FirstSourceMask;
+            *_VbDest                   = (*_VbDest & _FirstDestMask) | _FirstSourceVal;

```

Comment:
I'm not sure whether to extract `_FirstSourceVal`, like the shifting branches, or not, like the runtime aligned branch.

Reply:
IMO extracting is clearer.